README.md

```diff
@@ -1,2 +1,61 @@
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # mobile-version-portfolio-project
-The mobile version of a portfolio website.
\ No newline at end of file
+
+> The mobile version of my portfolio website
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+
+### Install
+https://github.com/Ogaga01/mobile-version-portfolio-project.git
+cd mobile-version-portfolio-project
+
```

Comment:
- [ ] Please update the "Getting Started" section content like how to install and run the project locally. Please work on that. :writing_hand:

Reply:
@mahbubul14 Can you please explain this further. I don't understand this